spec/features/user_can_delete_stations_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe "User can delete station" do
+  it "and click on button" do
+
+    Station.create!(name: "Galvanize", dock_count: 10, city: "Denver", installation_date: "11/08/17")
+
+    Station.create!(name: "Turing", dock_count: 10, city: "Denver", installation_date: "11/08/17")
+
+    visit '/stations/1'
+    click_on("delete")
+
+    expect(current_path).to eq('/stations')
```

Comment:
I also like the idea of testing the non-deleted station is still there. Will add a few `has_content` assertions to make it more comprehensive.